learning-mode.css

```diff
@@ -0,0 +1,282 @@
+:root {
+	--lm-header-height: 116px;
+	--border-color: var(--wp--preset--color--primary);
+}
+
+body {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+	---border-color: var(--border-color, red)
+}
+
+.sensei-course-theme-course-progress-bar {
+	--border-color:  var(--border-color)
+}
+
+.sensei-lms-course-navigation-lesson {
+	padding-top: 20px;
+	padding-bottom: 0;
+}
+
+.sensei-course-theme__sidebar {
+	--sidebar-width: 280px;
+}
+
+.sensei-course-theme__frame.sensei-course-theme__header {
+	padding-top: 45px;
+}
+
+.sensei-lms-course-navigation-lesson__title {
+	font-family: var(--wp--preset--font-family--system);
+	color: var(--wp--preset--color--primary);
+	font-size: 1rem;
+	font-weight: normal;
+}
+
+.sensei-lms-course-navigation-module__title {
+	font-family: var(--wp--preset--font-family--heading);
+	font-size: var(--wp--preset--font-size--medium);
+	line-height: 100%;
+	letter-spacing: 0.01em;
+	text-transform: uppercase;
+	color: var(--wp--preset--color--foreground);
+	text-decoration: none;
+}
+
+.sensei-lms-course-navigation-lesson__extra {
+	font-size: 1rem;
+    font-family: var(--wp--preset--font-family--system);
+    line-height: 100%;
+    font-weight: normal;
+    opacity: 1;
+    letter-spacing: 0.02em;
+}
+
+.sensei-lms-course-navigation-module__summary {
+	font-family: var(--wp--preset--font-family--system);
+	font-size: 11px;
+	font-weight: 400;
+	margin-top: 6px;
+  line-height: 16px;
+  opacity: 1;
+}
+
+.sensei-lms-course-navigation-module__header{
+	padding-top: 0;
+}
+
+.sensei-lms-course-navigation-module:not(:first-of-type) {
+	margin-top: 18px;
+}
+
+.wp-block-sensei-button:where(:not(.is-style-link)) a:hover{
+	text-decoration: none;
+}
+
+/*
```

Comment:
When a button is focused, it shrinks:

![Screen Recording 2022-11-22 at 21 03 11](https://user-images.githubusercontent.com/176949/203410953-f1442b99-c3ff-4cd5-a8f8-9e117019ebca.gif)

Reply:
Good find! Fixed it [here](https://github.com/Automattic/course/pull/68/commits/21dea191eea4d725f2c80ef9e9aacb469f0c8e58)